flask_app/static/js/upload.js

```diff
@@ -48,8 +48,21 @@ $(document).ready(function() {
         delay: 100,
         cache: true,
       },
+      templateResult: function(state) {
+        let query = $.trim($searchField.val());
+        let text = wrapSubstring(
+          state.text,
+          query,
+          '<span class="select2-highlight">',
+          '</span>',
+        );
+
+        return $('<div>' + text + '</div>');
+      }
```

Comment:
Minor improvement: highlight search string in completion dropdown. The color might be a bit too bright, will experiment.